src/applications/pensions/components/SectionField.jsx

```diff
@@ -2,12 +2,18 @@ import React from 'react';
 import { formatSSN } from 'platform/utilities/ui';
 
 export const convertDateFormat = date => {
-  const [year, month, day] = date.split('-');
-  return `${month}/${day}/${year}`;
+  if (date) {
```

Comment:
Why would we want to return date in this context since it's falsy? Can we guard the invoking call instead?

Reply:
Same here I refactored